kusto/ingest/ingest_example_test.go

```diff
@@ -6,26 +6,14 @@ import (
 
 	"github.com/Azure/azure-kusto-go/kusto"
 	"github.com/Azure/azure-kusto-go/kusto/ingest"
-	"github.com/Azure/go-autorest/autorest/azure/auth"
 )
 
 func ExampleIngestion_FromFile() {
 	var err error
 
-	authConfig := auth.NewClientCredentialsConfig("clientID", "clientSecret", "tenantID")
-	/*
-		Alteratively, you could so something like:
-		authorizer, err := auth.NewMSIConfig().Authorizer()
-		or
-		authorizer, err := auth.NewAuthorizerFromEnvironment()
-		or
-		auth.New...()
+	kcsb, _ := kusto.BuildConnectionStringWithAadApplicationCredentials("endpoint", "clientID", "clientSecret", "tenantID")
```

Comment:
The name of the method is long , may be shorten it to WithAadApplicationCredentials